### PR TITLE
[NOJIRA] Clarify programming language suggestion in Restaurant API README

### PR DESCRIPTION
**Pull request by agutteridge**

### CODE REVIEW

**Review by kareemsait — APPROVED** [review]
LGTM

File: SimpleRestaurantApi.md [hunk 1]
```diff
@@ -22,12 +22,13 @@ In general, consider your final solution to be a "production-ready" application,
...
-ability to pick up a new language, understand its basics, and apply that understanding in a short amount of time.  
-Picking a new language and showing that you can write some basic code using the strengths of that language, 
+Please don't fret too much about a lack of experience in the programming language specified in the Job Description. 
+Regard this as a chance to showcase your ability to pick up a new language, understand its basics, and apply that understanding 
+in a short amount of time. Picking a new language and showing that you can write some basic code using the strengths of that language, 
 even if the code isn’t mastery-level and doesn’t include super high-powered features, is a powerful showcase of your 
 software engineering skills.  In addition, other basic software engineering skills will be on display regardless of your
-familiarity with the actual programming language used for implementation.
+familiarity with the actual programming language used for implementation. Alternatively, you may use a language you are familiar with 
+as long as it isn't too obscure!
```
Comment (Aravindh-Sridaran-Paidy) [inline SimpleRestaurantApi.md:31]: `too obscure` is subjective, but we can deal with that when we get a submission thats too obscure for us but not for the candidate.

**Review by Aravindh-Sridaran-Paidy — APPROVED** [review]

File: SimpleRestaurantApi.md (hunk 1 above)
Comment (agutteridge) [reply SimpleRestaurantApi.md:31]: right! it might be better to have a list of accepted languages but we should confirm with everyone what these are 🙇

Comment (Gregor-Robertson-Paidy) [reply SimpleRestaurantApi.md:31]: Maybe just whatever we use in Paidy? 

- Java
- Scala
- Rust
- Python
- (We use Go, but I would really rather not put that in the list)

Comment (agutteridge) [reply SimpleRestaurantApi.md:31]: I think BNPL teams would really struggle if we received a Rust solution! We could phrase it like this:
* Language used in the relevant Paidy Team
* Java
* Python
?

Comment (Gregor-Robertson-Paidy) [reply SimpleRestaurantApi.md:31]: Should be fine. Not sure what others think though, so would be good to get someone else's opinion.

**Review by geekity — APPROVED** [review]

**Pull request merged**